specification/databricks/resource-manager/Microsoft.Databricks/preview/2021-04-01-preview/examples/DisableEncryption.json

```diff
@@ -0,0 +1,130 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg",
+    "workspaceName": "myWorkspace",
+    "api-version": "2018-04-01",
```

Comment:
>2018-04-01 [](start = 20, length = 10)

Make sure to update your api-version

Reply:
Fixed all the examples as well to use latest API version